src/components/VmStatus/VmStatus.js

```diff
@@ -21,62 +20,39 @@ import {
   getVmStatus,
 } from '../../utils/status/vm';
 import { getId, getVmImporterPods } from '../../selectors';
+import { Status, LinkStatus } from '../Status/Status';
```

Comment:
just `../Status` is enough, let's make use the flexibility given by of the webpack's resolver.